consul-lambda/consul-lambda-registrator/delete_event.go

```diff
@@ -79,6 +82,7 @@ func (e DeleteEvent) writeOptions() *api.WriteOptions {
 	return writeOptions
 }
 
+// AddAlias prepends an alias string to the DeleteEvent Name field
```

Comment:
While this comment is true, the full purpose of this func is to return a new `DeleteEvent` for a Lambda function with an alias so that it will also be removed from Consul when `Reconcile` is called.

Reply:
Thanks, that's great context!